gen/go/thetan/shared/v1/customized.pb.go

```diff
@@ -0,0 +1,292 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
+// versions:
+// 	protoc-gen-go v1.31.0
+// 	protoc        (unknown)
+// source: thetan/shared/v1/customized.proto
+
+package thetan_shared_v1
+
+import (
+	_ "github.com/srikrsna/protoc-gen-gotag/tagger"
+	protoreflect "google.golang.org/protobuf/reflect/protoreflect"
+	protoimpl "google.golang.org/protobuf/runtime/protoimpl"
+	anypb "google.golang.org/protobuf/types/known/anypb"
+	reflect "reflect"
+	sync "sync"
+)
+
+const (
+	// Verify that this generated code is sufficiently up-to-date.
+	_ = protoimpl.EnforceVersion(20 - protoimpl.MinVersion)
+	// Verify that runtime/protoimpl is sufficiently up-to-date.
+	_ = protoimpl.EnforceVersion(protoimpl.MaxVersion - 20)
+)
+
+// Customized dùng để gắn vào các item để cho phép app Thetan Creator có thể lưu được bản vẽ.
+type Customized struct {
+	state         protoimpl.MessageState
```

Comment:
mấy field này nó có lưu zô db k nhỉ :)))

Reply:
hông anh, nó là `unexported` mừ sao lưu vào đc @tuna-wolffun

Reply:
`The Go Driver only marshals and unmarshals exported fields.`

Ref: https://www.mongodb.com/docs/drivers/go/current/fundamentals/bson/#struct-tags
@tuna-wolffun